formats/properties/commonMain/src/kotlinx/serialization/Properties.kt

```diff
@@ -217,8 +217,7 @@ public class Properties(context: SerialModule = EmptyModule) : AbstractSerialFor
 
     public companion object {
         /**
-         * A [Properties] instance which
-         * does not have any [SerialModule]s installed.
+         * A [Properties] instance which does not have any [SerialModule] installed.
          */
         public val DEFAULT: Properties = Properties()
```

Comment:
The default instance is left deliberately: Properties have (in fact) no configuration and not state, neither it has common denominator like `StringFormat`

Reply:
Ok, works for me